### PR TITLE
MULTIARCH-4242 - Deprecating sdn network deployment jobs on ARM in 4.15+

### PR DESCRIPTION
**Pull request by tvardema**

### CODE REVIEW

File: clusters/app.ci/release-reimport-controller/deployment.yaml [hunk 1]
```diff
@@ -29,6 +29,7 @@ spec:
       - command:
         - /usr/bin/release-reimport-controller
         - start
+        - --dry-run
```
Comment (bradmwilliams) [inline clusters/app.ci/release-reimport-controller/deployment.yaml:32]: You'll need to rebase against the latest/greatest changes.  This parameter was removed recently.

**Review by bradmwilliams — CHANGES_REQUESTED** [review]

File: clusters/app.ci/release-reimport-controller/deployment.yaml (hunk 1 above)
Comment (tvardema) [reply clusters/app.ci/release-reimport-controller/deployment.yaml:32]: I literally just generated that an hour ago... talk about unfortunate timing, ha ha ha

Comment (bradmwilliams) [reply clusters/app.ci/release-reimport-controller/deployment.yaml:32]: This isn't your fault actually...  It needs to be corrected in the correct place...
https://github.com/openshift/release/pull/49343
That will need to merge and then this file should drop out after you rebase and do a:
`make release-controllers`

Comment (bradmwilliams) [reply clusters/app.ci/release-reimport-controller/deployment.yaml:32]: Sorry for the churn!

**Pull request merged**